### PR TITLE
Tython Blog: Permissions Assistant User Analysis

### PR DESCRIPTION
**Pull request by ndrewr**
First draft of User Analysis blog post.

I was trying to ride with a recipe/chef metaphor, but faltered here and there 😂 Hopefully still succinct.

Side-note: Thinking to revisit this post and add links to subsequent articles as they are published.

### CODE REVIEW

File: _drafts/analyzing-users-with-tython-permissions-assistant.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+keywords: ['permissions', 'permissions assistant', 'admin', 'tools', 'user analysis']
+slug: 
+---
+
+[Andrew Chen](https://www.linkedin.com/in/ndrewr/)
+<br/>
+
+### A Recipe for Frustration
+
+Managing permissions in Salesforce has for years (decades?) been a thorn in the side of admins. Getting to the bottom of "Who can see what and Why" is, like my scrambled eggs recipe, harder than it has any right being. The team at Tython set out to build a tool that assists in getting permissions under control. Enter Tython Permissions Assistant (currently in beta).
```
Comment (ndrewr) [inline _drafts/analyzing-users-with-tython-permissions-assistant.md:16]: Not sure if we need that "beta" note.

Comment (scottbcovert) [reply _drafts/analyzing-users-with-tython-permissions-assistant.md:16]: I don't think we do, truthfully

Comment (scottbcovert) [reply _drafts/analyzing-users-with-tython-permissions-assistant.md:16]: I don't think we do, truthfully

File: _drafts/analyzing-users-with-tython-permissions-assistant.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+    <img src="/images/2023-12-29-pa-user-analysis-compare-user-diffs.png" alt="Compare user permission differences" title="Comparing user permission differences" style="max-width:700px;"/>
+    <p style="color: #6a6a6a; font-size: .75rem;">Viewing only the different permission assignments between two users.</p>
+</div>
+<br/>
+
+### Master Chef
+
+Tython Permissions Assistant User Analysis brings order to a chaotic kitchen. It lays out user access neatly on the counter. Combined with Permissions Assistant's other tools, admins can really get cooking!
+
+Permissions Assistant will be available on the App Exchange in early 2024. And we are just getting started.
```
Comment (scottbcovert) [inline _drafts/analyzing-users-with-tython-permissions-assistant.md]: Perhaps we can move the "beta" note here:

`Permissions Assistant is currently in invite-only beta, but will be available on the AppExchange in 2024.`

File: _drafts/analyzing-users-with-tython-permissions-assistant.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+
+The team at Tython is always open to feedback! Interested in what we have in the oven? Some other functionality you would like to see? [Drop us a line, anytime](mailto:support@tython.co?subject=Permissions%20Assistant%20Demo)!
```
Comment (scottbcovert) [inline _drafts/analyzing-users-with-tython-permissions-assistant.md]: If we add the invite-only beta note above then perhaps we could also reference that in the CTA--"Want to request access to the Permissions Assistant beta?"

**Review by scottbcovert — APPROVED** [review]
I"m really interested to hear about this complicated recipe you have for scrambled eggs now...

**Pull request merged**